src/components/cards-grid-list/index.tsx

```diff
@@ -64,7 +64,9 @@ function CardsGridList({
 				} as CSSProperties
 			}
 		>
-			{children}
+			{Children.map(children, (child) => {
+				return <li>{child}</li>
```

Comment:
💭 Does this need a key?

Reply:
I don't think needs `key` - my understanding is `Children.map` will use the `key` of each `child`. My main point of reference was [this StackOverflow question](https://stackoverflow.com/a/50303840), there's also [an example in the beta React docs](https://beta.reactjs.org/apis/react/Children#transforming-children) that I think kind of confirms this.